Incorrect usage of generics in typescript definitions

TypeScript build fails due to incorrect type definitions. Not sure how this was possible in the first place, but I can only assume that the TypeScript compiler was more lenient with generics at some point.

#### Which package is this issue related to?

ffe-buttons-react
ffe-form-react

#### Describe your issue (screenshots welcome!)

```Shell
node_modules/@sb1/ffe-buttons-react/types/index.d.ts:6:26 - error TS2304: Cannot find name 'T'.

6     innerRef?: React.Ref<T>;
```

#### What is the expected behavior?

We need to declare the type parameter:

Incorrect:
```TypeScript
export interface MinimalBaseButtonProps extends React.HTMLProps<HTMLElement> {
```

Correct
```TypeScript
export interface MinimalBaseButtonProps<T> extends React.HTMLProps<HTMLElement> {
```

#### What is the actual behavior?

Build fails.

#### Steps to reproduce

Import ffe-buttons-react into any project using TypeScript 3.7 and compile using `tsc`.

--- a/packages/ffe-dropdown-react/src/index.d.ts
+++ b/packages/ffe-dropdown-react/src/index.d.ts
@@ -6,7 +6,7 @@ export interface DropdownProps
     className?: string;
     inline?: boolean;
     dark?: boolean;
-    innerRef?: React.Ref<T>;
+    innerRef?: React.Ref<HTMLSelectElement>;
 }
 
 declare class Dropdown extends React.Component<DropdownProps, any> {}

